rundeck_exporter.py

```diff
@@ -375,7 +375,11 @@ def collect(self):
         Rundeck system info
         """
         system_info = self.request_data_from('/system/info')
-        api_version = int(system_info['system']['rundeck']['apiversion'])
+        try:
+             api_version = int(system_info['system']['rundeck']['apiversion'])
+        except ValueError as err:
+              pass
```

Comment:
I think that setting a `try...except` block ignoring the exception could cause an unexpected behavior like `api_version was not set`. A problem that could be seen here is a `KeyError`, for example, if for some reason the Rundeck guys change the attribute names.